docs/content/direct/contributor.md

```diff
@@ -1,6 +1,6 @@
 # KubeStellar Contributors
 
-![](./images/construction.png){: style="height:100px;width:100px"}

```

Comment:
Yeah, that has not been rendering well when viewed directly as a source on GitHub (see the curly braced stuff shown at the bottom):

<img width="993" alt="Screenshot 2024-05-01 at 3 32 45 PM" src="https://github.com/kubestellar/kubestellar/assets/14296719/f1163acf-ac06-472a-929c-91bdb1793eee">